lib/sles4sap_publiccloud.pm

```diff
@@ -424,26 +426,61 @@ sub wait_for_pacemaker {
 }
 
 =head2 setup_sbd_delay
-     setup_sbd_delay([set_delay => $set_delay]);
+     $self->setup_sbd_delay();
 
-     Set (activate or deactivate) SBD_DELAY_start setting in /etc/sysconfig/sbd.
+     Set (activate or deactivate) SBD_DELAY_START setting in /etc/sysconfig/sbd.
      Delay is used in case of cluster VM joining cluster too quickly after fencing operation.
      For more information check sbd man page.
 
+     Setting is changed via OpenQA parameter: HA_SBD_START_DELAY
+     Possible values:
      "no" - do not set and turn off SBD delay time
      "yes" - sets default SBD value which is calculated from a formula
-     "<number of seconds>" - sets sepcific delay in seconds
+     "<number of seconds>" - sets specific delay in seconds
+
+     Returns integer representing wait time.
 
 =cut
 
 sub setup_sbd_delay() {
-    my ($self, $set_delay) = @_;
-    my $delay = $set_delay || "no";
-    record_info("SBD delay", "Setting SBD delay to: $delay");
+    my ($self) = @_;
+    my $delay = get_var('HA_SBD_START_DELAY');
+    $delay =~ s/(?<![ye])s//g;
+    if (get_var('HA_SBD_START_DELAY') eq '') {
```

Comment:
Why `get_var('HA_SBD_START_DELAY')` and not `$delay`?

Reply:
I am not sure what does get_var() return if the variable is not defined. I was afraid that it will return 0 instead of empty variable or undef and in that case function would change sbd value to literal 0. 
But now that I have unit tests I can check it in to time !

Will be back :D

Reply:
And it works. Added some more values to the unit test as well.

Reply:
`get_var()` takes 2 arguments, the name of the setting, and a default value. If no default value is provided, and the setting is not defined, then it will return `undef`.